life-and-letters/modernized/09-chapter-8.adoc

```diff
@@ -4584,14 +4592,21 @@ and in particular to prevent the Indians from becoming engaged in the war.
 These efforts were frequent and strenuous, but proved unavailing.
 In his Journal are the following entries:
 
-"`Fourth month 9th, 1756.--Had some labor among Friends,
+[.embedded-content-document]
+--
+
+Fourth month 9th, 1756.--Had some labor among Friends,
 to endeavor to prevent a cruel Indian war;
 and had also a conference with the present and late Governor, along with John Pemberton,
 upon the present posture of affairs; they received us with candour,
-but our labor was ineffectual, for on the "`10th,
+but our labor was ineffectual, for on the
```

Comment:
So this is a little weird. I didn't know if I should just keep these journal entries together, but I put them how they are in the original because on line 4593 it says "entries" and not "entry". But I am probably over-thinking it. 😄